layouts/partials/head.html

```diff
@@ -32,6 +32,6 @@
 		<link rel="apple-touch-icon" sizes="180x180" href="{{ "images/apple-touch-icon.png" | relURL }}">
 	
 		<!-- RSS -->
-		<link href="{{ .RSSLink }}" rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" />
+		<link href="{{ .RelPermalink }}" rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" />
```

Comment:
The error message states that `.RSSLink` should use `{{ with .OutputFormats.Get "RSS" }}{{ . RelPermalink }}{{ end }}` instead.